ddtrace/internal/utils/http.py

```diff
@@ -209,3 +210,13 @@ def w3c_encode_tag(args):
     tag_val = pattern.sub(replacement, tag_val)
     # replace = with ~ if it wasn't already replaced by the regex
     return tag_val.replace("=", "~")
+
+
+def convert_128bit_traceid_to_64bit(trace_id):
```

Comment:
since the other helper is getting the higher order 64-bits, should this be named to get the lower order 64-bits ?